sky/clouds/gcp.py

```diff
@@ -424,6 +424,21 @@ def make_deploy_resources_variables(
         # issue when first booted.
         image_id = 'skypilot:cpu-debian-11'
 
+        def _failover_disk_tier() -> Optional[resources_utils.DiskTier]:
+            if (r.disk_tier is not None and
+                    r.disk_tier != resources_utils.DiskTier.BEST):
+                return r.disk_tier
+            # Failover disk tier from high to low.
+            all_tiers = list(reversed(resources_utils.DiskTier))
+            start_index = all_tiers.index(GCP._translate_disk_tier(r.disk_tier))
```

Comment:
What does this `_translate_disk_tier` do?

Reply:
Convert `best` and `None` to corresponding tier